libraries/core_libs/network/src/tarcap/packets_handlers/pbft_sync_packet_handler.cpp

```diff
@@ -73,6 +73,7 @@ void PbftSyncPacketHandler::process(const PacketData &packet_data, const std::sh
   if (sync_block.pbft_blk->getPeriod() != pbft_mgr_->pbftSyncingPeriod() + 1) {
     LOG(log_dg_) << "Block " << pbft_blk_hash << " period unexpected: " << sync_block.pbft_blk->getPeriod()
                  << ". Expected period: " << pbft_mgr_->pbftSyncingPeriod() + 1;
+    restartSyncingPbft(true);
```

Comment:
don't we need to disable syncing by `syncing_state_->set_pbft_syncing(false);`

Reply:
No, since restartsyncing with force flag will set it to true or false inside